assets/css/sidebar.css

```
@@ -109,6 +109,21 @@
   display: none;
 }
 
+.sidebar .sidebar-staleVersion {
+    display: inline;
+}
+
+.sidebar .sidebar-staleVersion > a {
+    color: var(--sidebarStaleVersion);
+    font-weight: 400;
+}
+
+.sidebar .sidebar-staleIcon {
+    font-size: 18px;
+    position: relative;
+    top: 3px;
```

```suggestion
    top: 3px;
    line-height: 0;
```

This is required so the caret on the left-side preserves its alignment. :)